crypto/pedersen/pedersen.go

```diff
@@ -0,0 +1,56 @@
+package pedersen
+
+import (
+	"encoding/hex"
+	"fmt"
+	"hash"
+	"os"
+	"os/exec"
+)
+
+// Requires to follow instructions to install pycrypto: https://github.com/stefandeml/zokrates-pycrypto
+// And either set CENT_PED_BIN with cli.py location or store the repo in $HOME/zksnarks/pycrypto
+
+// NewPedersenHash returns a pedersen hash function
+func NewPedersenHash() hash.Hash {
+	// Resolve python binary location
+	pedBin := os.Getenv("CENT_PED_BIN")
+	if pedBin == "" {
+		pedBin = fmt.Sprintf("%s/zksnarks/pycrypto/cli.py", os.Getenv("HOME"))
```

Comment:
IMO, I think we should ship the python libs into the docker so that we can properly version them. That way we can always know and not expect anything from the env and wont break anything when we move to the native go implementation except for the docker which will be self contained